pkg/storaged/dialog.jsx

```diff
@@ -1175,13 +1178,15 @@
 }
 
 export const StopProcessesMessage = ({ mount_point, users }) => {
+    const mock = useMock()?.busy;
+
     const process_rows = users.filter(u => u.pid).map(u => {
         return {
             columns: [
-                u.pid,
+                mock?.pid ?? u.pid,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17654-20230322-115306-eab6c63e-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)